### PR TITLE
changes

### PR DESCRIPTION
**Pull request by florinaToldea**

### CODE REVIEW

File: GreenhouseNode/GreenhouseNode/revision/DownLink.c [hunk 1]
```diff
@@ -0,0 +1,65 @@
+/*
```
Comment (jhviggo) [inline GreenhouseNode/GreenhouseNode/revision/DownLink.c:1]: Why is the folder called `revision`? I'd prefer just putting the uplink functionality into the existing `LoRaWAN.c`

File: GreenhouseNode/GreenhouseNode/revision/DownLink.c [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+		
+		if((downLink_payLoad.bytes[0] << 8) + downLink_payLoad.bytes[1] ==0)
+		{
+			rc_servo_setPosition(0, -100);
+		}
+		else
+		{
+			rc_servo_setPosition(0, 100);
+		}
+	}
```
Comment (jhviggo) [inline GreenhouseNode/GreenhouseNode/revision/DownLink.c:64]: What is all of this used for?

File: GreenhouseNode/GreenhouseNode/revision/UpLink.c [hunk 3]
```diff
@@ -0,0 +1,167 @@
+/*
```
Comment (jhviggo) [inline GreenhouseNode/GreenhouseNode/revision/UpLink.c:1]: I've already made the LoRa uplink in the `LoRaWAN.c` file.

**Comment by jhviggo** [comment]
Overall I think you should just 
* remove the uplink implementation
* move the downlink to the parent folder 
* then rename it so it fits the class diagram.

**Comment by OZoneSQT** [comment]
It looks an outdated pull request

**Pull request closed without merging**